data/core/encyclopedia/geography.cfg

```diff
@@ -99,8 +105,8 @@ Over the River Aethen, south of Fort Tahn, is a Wesnothian frontier region. It i
     • Soradoc: The northernmost border outpost of Wesnoth, controls the confluence of the Weldyn River and the Great River.
     • Fort Tahn: The southernmost border outpost, controls the north/south road crossing the River Aethen.
     • Tath: Important fort city north of Dan’Tonk, exerts control over the wilderness country around the east of the Brown Hills and north to the Ford of Abez.
-    • Westin: The capital and largest city of the southern Wesnothian province of Kerlath. The marvelous Citadel of Westin can be found there.
-
+    • Westin: The capital and largest city of the southern Wesnothian province of Kerlath. The marvelous Citadel of Westin can be found there." + "
+    
```

Comment:
Should avoid adding unnecessary white-space like this.

Reply:
IIUC, you mean the trailing spaces / trailing tabs on the line, not the blank line itself?

Reply:
Yes.